docs/product-installation-guide/src/main/asciidoc/clustering-con.adoc

```diff
@@ -1,15 +1,64 @@
 [id='clustering-con']
-= Clustering with {PRODUCT} for design-time development environments
+= Clustering  {PRODUCT} 
+
+By clustering two or more computers, you have the benefits of high availability, enhanced collaboration, and load balancing. High availability decreases the chance of a loss of data when a single computer fails because data on all of the computers (nodes) of the cluster is replicated on each node of the cluster. When a computer fails, another computer fills the gap by providing a copy of the data that was on the failed computer. When the failed computer comes online again, it resumes its place in the cluster. Load balancing shares the computing load across the nodes of the cluster. Doing this improves the overall performance of the cluster.
+
+There are several ways that you can cluster {PRODUCT} components. This document describes how to cluster the following scenarios:
+
+* Clustering a {PRODUCT} development environment
+* Clustering a {KIE_SERVER} standalone runtime environment
+* Clustering a {KIE_SERVER} runtime environment that is managed by the standalone {KIE_SERVER} controller
+
+== Clustering {PRODUCT} in a development environment
+Developers use {PRODUCT} it create rules and processes that users then run in a runtime environment to assist with decision making. The primary benefit of clustering {PRODUCT} development environments is high availability and enhanced collaboration. For example, if clustering is configured and Developer X is authoring rules on {CENTRAL} Node1 then Developer Y, who is working on {CENTRAL} Node2, can see the rules authored by Developer X as they are created. Furthermore, all of the developers working on individual nodes of the cluster will see the same synchronized content. In addition, clustering provides high availability in the rules development environment. If Developer X is working on Node1 and that node fails, Developer X's work is preserved and visible on any other node of the cluster.
+
+Most development environments consist of {CENTRAL} for creating rules and processes and at least one {KIE_SERVER} to test those rules and processes. 
 
 [NOTE]
 ====
-This section is specifically intended for {CENTRAL} development environments where you want to cluster the Git repository. It is not necessary to create a clustered environment for {KIE_SERVER} production environments.
+You can also use this configuration for runtime environemtns if you want to use {CENTRAL} to manage the {KIE_SERVER}s in the cluster.
 ====
 
-Red Hat recommends that you consider clustering two or more computers to create a high availability clustered environment for design-time development environments. Doing this enhances collaboration and provides high availability.
+.Prerequiste
+* A Git repository on an NFS server is available.
```

Comment:
It would be enought to say that: 'A NFS server is available, with a monted partition accessible to EAP user'